components/pages/file-repository/FileTable/TsvDownloadButton.tsx

```diff
@@ -152,19 +155,27 @@ export default ({
       css={css`
         margin-right: 8px;
         width: 160px;
+        :disabled {
+          background-color: #f2ecfd;
+          color: #523785;
+        }
       `}
       variant="secondary"
       size="sm"
       onItemClick={onItemClick}
       menuItems={menuItems}
+      isLoading={loading}
+      loadWithChildren={true}
```

Comment:
```suggestion
      loadWithChildren
```

setting a boolean prop to `true` is redundant

Reply:
kk thanks, updated